target/linux/ipq40xx/base-files/lib/upgrade/sony.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
```

Comment:
No shebang for /lib files please (and no execute bit, I haven't checked if there is one ...)

Reply:
Thank you, I forgot it and I'll drop shebang. Currently, sony.sh has no execute bit and I think it's OK.